### PR TITLE
VACMS-18186: CLP GA update to remove custom analytics from va-links

### PR DESCRIPTION
**Pull request by chriskim2311**
## Summary

The va-link component has been enhanced to provide destination and can be used now for applicable Campaign Landing Page features, along with removal of the GA custom events.

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18186

## Testing done
Testing done in review instance at urls listed below

- [ ] **Element with custom events:** (What you can do) "Learn how to get your flu shot" link
/initiatives/covid-flu
- [ ] **Element with custom events:** (Spotlight) "Why is VA helping Veterans to vote?" link
/initiatives/vote/
- [ ] **Element with custom events:** (Stories) "Staying safe when considering employment with a foreign organization" link
/initiatives/protecting-veterans-from-fraud/
- [ ] **Element with custom events:** (VA Benefits) "VA health benefits" link
/initiatives/protecting-veterans-from-fraud/

## Screenshots
(What you can do) "Learn how to get your flu shot"
Cursor_and_Flu_And_COVID-19__One_Visit__Two_Vaccines___Veterans_Affairs

(Spotlight) "Why is VA helping Veterans to vote?"
How_Veterans_Can_Register_To_Vote___Veterans_Affairs

(Stories) "Staying safe when considering employment with a foreign organization" 
Protecting_Veterans_From_Fraud___Veterans_Affairs

(VA Benefits) "VA health Benefits" 
Protecting_Veterans_From_Fraud___Veterans_Affairs

## What areas of the site does it impact?

CLP Pages

## Acceptance criteria
- [ ] a11y Review

### CODE REVIEW

File: src/site/layouts/campaign_landing_page.drupal.liquid [hunk 1]
```diff
@@ -310,14 +299,13 @@
               <div class="vads-u-background-color--gray-light-alt vads-u-height--full vads-u-padding--2 vads-u-display--flex vads-u-flex-direction--column vads-u-justify-content--space-between medium-screen:vads-u-margin-x--1 medium-screen:vads-u-margin-y--0">
                 <h3 class="vads-u-margin--0">{{ resource.entity.name }}</h3>
                 <p class="vads-u-flex--1">{{ resource.entity.fieldDescription }}</p>
-                <a
+                <va-link
```
Comment (randimays) [inline src/site/layouts/campaign_landing_page.drupal.liquid:302]: I think, for accessibility, we'd want to add `filetype="PDF"` and remove the `(PDF)` from the `text` attribute. @laflannery please correct me if not.

**Comment by laflannery** [comment]
@randimays Does this Download link have to be the `va-link` component? I'm not sure exactly what you all are doing with the analytics piece of this. 

However, from an a11y standpoint I would prefer to leave this as is because of the existing `aria-label`. The component doesn't yet have this available so if we were to change this we would be removing the unique links.

**Comment by chriskim2311** [comment]
@laflannery We are converting <a> to va-link where applicable since they have added better built in analytics. But I can remove this change for the download links if that would be a better experience for veterans.

**Comment by randimays** [comment]
@laflannery Thank you; sorry to keep pulling you in on this! It's hard to keep track of the state of `va-link` and when to use it 😮‍💨 The `va-link` does accept the `aria-label` attribute and read it aloud; I'm not sure what part of the `aria-label` is not usable.  But I do see the `dfn` element is still in there when the `filetype` is added, so we shouldn't use `filetype`. 

Is it not possible to use the `va-link` as Chris has it in the PR currently?

```
<va-link
  aria-label="Download {{ resource.entity.name }} PDF"
  download
  href="{{ resource.entity.fieldMediaExternalFile.uri }}"
  text="Download (PDF)"
>
</va-link>
```

@chriskim2311 Here's a recent discussion about `filetype`/`dfn`: https://github.com/department-of-veterans-affairs/content-build/pull/2100#issuecomment-2135995793

**Comment by laflannery** [comment]
Brief recap:
- We can't use `aria-label` on the `va-link` component. It's not a supported prop and the fix for this has not yet been released from DST.
- Because of that and because we don't want to hold this for them, I would like to change the download links back to the standard HTML `<a>` tags so that these can remain unique, accessible links.

[Link to a slack thread](https://dsva.slack.com/archives/C06NMUK12LR/p1718729009533489) that had a bit more discussion

**Comment by chriskim2311** [comment]
@randimays @laflannery Thanks for the insight on the va-link usage. I have reverted the changes made to the download links and this is ready for review thanks!

**Review by randimays — APPROVED** [review]
LGTM!

**Comment by laflannery** [comment]
Looks good!

**Pull request merged**